### PR TITLE
Bugfix/fix undefined template variables

### LINKED ISSUE #504: Does not set {{ title }} context variable
The default ``django/contrib/admin/templates/admin/base_site.html``, is used by the page that allows you to select the page format - looks like this is ``export.html``. It has:

```
{% block title %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
```

Unfortunately, this project does not supply a value of title to this template, resulting in incorrect output or (in my case) fatal errors as I have configured Django to generate a fatal error if a template variable is not defined.

### PR DESCRIPTION
**Pull request by brianmay**
This fixes #504.

### CODE REVIEW

File: import_export/admin.py [hunk 1]
```diff
@@ -266,6 +266,7 @@ def import_action(self, request, *args, **kwargs):
         elif django.VERSION >= (1, 7, 0):
             context.update(self.admin_site.each_context())
 
+        context['title'] = "Import"
```
Comment (bmihelac) [inline import_export/admin.py]: Can translation be used here? _('Import')

File: import_export/admin.py [hunk 2]
```diff
@@ -394,6 +395,7 @@ def export_action(self, request, *args, **kwargs):
         elif django.VERSION >= (1, 7, 0):
             context.update(self.admin_site.each_context())
 
+        context['title'] = "Export"
```
Comment (bmihelac) [inline import_export/admin.py]: _('Export')

**Comment by brianmay** [comment]
Updated to support translations.

**Comment by bmihelac** [comment]
Merged, thanks!

**Pull request merged**